metricbeat/module/apache/apache.asciidoc

```diff
@@ -15,5 +15,21 @@ The Apache Module has currently one MetricSet which is called status. It fetches
 === Apache Status Metricset
 
 
-TODO
-* List available selectors
+The apache status metricset offers the following selectors:
+
+* uptime
+* cpu
+* connections
+* load
+* scoreboard
```

Comment:
After further thought on this I think it will be better and more maintainable if we just link to the exported fields and explain that those fields can be used with an include_fields or drop_fields filter. Then we don't have to duplicate the fields or the descriptions.